Lib/test/test_ttk/test_widgets.py

```diff
@@ -1147,7 +1185,12 @@ def test_traversal(self):
 
         self.nb.select(0)
 
-        focus_identify_as = 'focus' if sys.platform != 'darwin' else ''
+        if sys.platform == 'darwin':
+            focus_identify_as = ''
+        elif sys.platform == 'win32':
+            focus_identify_as = 'focus'
+        else:
+            focus_identify_as = 'focus' if tk_version < (9,0) else 'padding'
         self.assertEqual(self.nb.identify(5, 5), focus_identify_as)
```

Comment:
Trying this locally on Linux with Tcl/Tk 9.0b3, I get a failure where 'focus' is gotten instead of 'padding'.  Maybe we should just adjust the test like the following?

```suggestion
        self.assertIn(self.nb.identify(5, 5), ['', 'focus', 'padding'])
```

Reply:
I think this is due to a recent change in the size of the focus ring.